server/routes/schedules.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const conn = require("../mariadb");
+const { StatusCodes } = require("http-status-codes");
+const {
+  getAllSchedules,
+  getScheduleById,
+  addSchedule,
+  updateSchedule,
+  deleteSchedule,
+} = require("../controller/schedulesController");
+
+router.use(express.json());
+
+router.get("/", getAllSchedules);
+router.get("/detail/:id", getScheduleById);
```

Comment:
api endpoint와 실제로 해당 함수의 매개변수를 통해서 어떤 값으로 찾는지를 알 수 있어서 이렇게 표현하기도 하는 편입니다~
요건 개인 취향에따라 고려해주시면 될 것 같아요!
```suggestion
router.get("/detail/:id", getScheduleBy);
```

Reply:
넵 감사합니다. 
getScheduleById가 눈에 익어서 여기선 getScheduleById 사용하겠습니다..!